BookingV2.Logic/Responses/ApiResponse.cs

```diff
@@ -0,0 +1,17 @@
+namespace BookingV2.Logic.Responses
+{
+    public class ApiResponse<T>
+    {
+        public bool IsSucceeded { get; set; } = true;
+
+        public T? Response { get; set; }
```

Comment:
please place this class to API level, cause this class represents server response